generators/app/index.js

```diff
@@ -255,6 +259,11 @@ module.exports = JhipsterGenerator.extend({
                 this.generatorType = 'server';
                 this.skipUserManagement = configOptions.skipUserManagement = true;
             }
+            if (this.applicationType == 'uaa') {
+                this.skipClient = true;
+                this.generatorType = 'server';
+                this.skipUserManagement = false;
```

Comment:
I would prefer `this.skipUserManagement = configOptions.skipUserManagement = false;` for consistency here